### PR TITLE
Theme changes for openpgp on dark theme

### PR DESCRIPTION
**Pull request by Valodim**
What it says on the tin.

(A todo I didn't do in this PR is to split up the colors by semantics and use appropriate names)

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/view/ThemeUtils.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+public class ThemeUtils {
+    private static final TypedValue TYPED_VALUE = new TypedValue();
+
+    @ColorInt
+    public static int getStyledColor(Context context, @AttrRes int attr) {
+        return getStyledColor(context.getTheme(), attr);
+    }
+
+    @ColorInt
+    public static synchronized int getStyledColor(Theme theme, @AttrRes int attr) {
```
Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/view/ThemeUtils.java]: Why `synchronized`? This should only ever be called from the main thread, right?

Comment (Valodim) [inline k9mail/src/main/java/com/fsck/k9/view/ThemeUtils.java]: I suppose. It didn't seem sufficiently obvious that this method isn't thread safe though so I wanted to do *something*. @UiThread annotation would be fine as well.

Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/view/ThemeUtils.java]: Let's use `@UiThread` then.

Comment (Valodim) [inline k9mail/src/main/java/com/fsck/k9/view/ThemeUtils.java]: :+1:

**Comment by cketti** [comment]
:+1:

**Pull request merged**